src/main/web_src/src/pages/gruppeOversikt/FinnPerson.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from 'react'
+//@ts-ignore
+import AsyncSelect from 'react-select/async'
+import { useAsyncFn } from 'react-use'
+import { Redirect } from 'react-router-dom'
+import useBoolean from '~/utils/hooks/useBoolean'
+import { TpsfApi } from '~/service/Api'
+import NavButton from '~/components/ui/button/NavButton/NavButton'
+import DollyModal from '~/components/ui/modal/DollyModal'
+import { Label } from '~/components/ui/form/inputs/label/Label'
+import { Søkeknapp } from 'nav-frontend-ikonknapper'
+import { AsyncFn } from 'react-use/lib/useAsync'
+
+type FinnPerson = {
+	naviger: Function
+}
+
+type Option = {
+	value: string
+	label: string
+}
+
+type Person = {
+	ident: string
+	fornavn: string
+	mellomnavn?: string
+	etternavn: string
+}
+
+type Respons = {
+	value: {
+		data: {
+			gruppe?: {
+				id: number
+			}
+			error?: string
+			message?: string
+		}
+	}
+}
+
+export default function FinnPerson({ naviger }: FinnPerson) {
+	const [isFinnModalOpen, openFinnModal, closeFinnModal] = useBoolean(false)
+	const [redirectToGruppe, setRedirect] = useBoolean()
+
+	const [ident, setIdent] = useState(null)
+	const [gruppe, setGruppe] = useState(null)
+	const [feilmelding, setFeilmelding] = useState(null)
+
+	const [options, fetchOptions]: AsyncFn<any> = useAsyncFn(async tekst => {
+		const { data }: any = await TpsfApi.soekPersoner(tekst)
```

Comment:
Siden `data.map` under gir personer, kunne dette kanskje vært `Array<Person>` i steden for `any`?

Reply:
Uff, ja det synes jeg også! Men TS er ikke enig... Forsøkte en stund å løse den forrige uke, men tror nesten jeg må gi opp 😞

Reply:
Ts knows best 🙏